### PR TITLE
Update Architecture.md with disperse volume info

### PR DESCRIPTION
**Pull request by Sheetalpamecha**
Signed-off-by: Sheetal <spamecha@redhat.com>

### CODE REVIEW

**Comment by sankarshanmukhopadhyay** [comment]
Is there a reviewer for a LGTM sign off?

File: docs/Quick-Start-Guide/Architecture.md
Comment (kshithijiyer) [inline docs/Quick-Start-Guide/Architecture.md:146]: Don't we need to add either disperse-data or redundancy here?

File: docs/Quick-Start-Guide/Architecture.md
Comment (kshithijiyer) [inline docs/Quick-Start-Guide/Architecture.md:127]: Don't we need to add either disperse-data or redundancy here?

**Review by kshithijiyer — CHANGES_REQUESTED** [review]

File: docs/Quick-Start-Guide/Architecture.md
Comment (Sheetalpamecha) [reply docs/Quick-Start-Guide/Architecture.md:146]: No. All the three options(disperse, disperse-data and redundancy) can be used independently as well as in combination with each other. So this will work perfectly fine.

File: docs/Quick-Start-Guide/Architecture.md
Comment (Sheetalpamecha) [reply docs/Quick-Start-Guide/Architecture.md:127]: Same as above comment

**Comment by sankarshanmukhopadhyay** [comment]
Amar, once you have a sign-off I'll go ahead and merge.

File: docs/Administrator Guide/Setting Up Volumes.md
Comment (aspandey) [inline docs/Administrator Guide/Setting Up Volumes.md:376]: This could be confusing and user might ask question "Do we need to have as many node as the number of bricks in a disperse volume, in this case 6 nodes?"
Just like an example of bad setup, we should also give an example of good setup, specially considering distributed disperse volume where 2 bricks can be hosted on same node if they belong to different subvol.

File: docs/Quick-Start-Guide/Architecture.md
Comment (aspandey) [reply docs/Quick-Start-Guide/Architecture.md:146]: It can be used independently, however, it is worth mentioning the default behaviour. I think if you just mention disperse count, it will calculate best redundancy count on its own (which is 50% I think).
So, it would be good to mention this and also recommend that user provide disperse-data and redundancy count as per their need.

**Comment by sankarshanmukhopadhyay** [comment]
@Sheetalpamecha could you look at the comments from @aspandey and decide on taking it forward.

File: docs/Administrator Guide/Setting Up Volumes.md
Comment (Sheetalpamecha) [reply docs/Administrator Guide/Setting Up Volumes.md:376]: Done

**Comment by Sheetalpamecha** [comment]
> @Sheetalpamecha could you look at the comments from @aspandey and decide on taking it forward.

Done

File: docs/Administrator Guide/Setting Up Volumes.md
Comment (Sheetalpamecha) [reply docs/Administrator Guide/Setting Up Volumes.md:376]: Addressed both the comments from ashish

File: docs/Quick-Start-Guide/Architecture.md
Comment (Sheetalpamecha) [reply docs/Quick-Start-Guide/Architecture.md]: Done

File: docs/Quick-Start-Guide/Architecture.md
Comment (Sheetalpamecha) [reply docs/Quick-Start-Guide/Architecture.md]: Done

File: docs/Quick-Start-Guide/Architecture.md
Comment (kshithijiyer) [inline docs/Quick-Start-Guide/Architecture.md:163]: I would suggest to put this in code.

File: docs/Quick-Start-Guide/Architecture.md
Comment (kshithijiyer) [inline docs/Quick-Start-Guide/Architecture.md:169]: Same as above

File: docs/Quick-Start-Guide/Architecture.md
Comment (Sheetalpamecha) [reply docs/Quick-Start-Guide/Architecture.md:163]: yes done.

File: docs/Quick-Start-Guide/Architecture.md
Comment (Sheetalpamecha) [reply docs/Quick-Start-Guide/Architecture.md:169]: Done

**Pull request merged**
